cypress/e2e/suggestions.cy.ts

```diff
@@ -0,0 +1,33 @@
+describe("suggestions page", () => {
```

Comment:
There are no tests here for edge cases such as a failing requests, the loading spinner or when there are no requests. I think that you should not only test that it works, but also test what happens when it does not work.

i.e. you could stimulate no responses with this:
`cy.intercept("/rest/v1/suggestion*", [])`